tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -139,8 +141,8 @@ RUN echo "hwloc_base_binding_policy = none" >> $OPEN_MPI_PATH/etc/openmpi-mca-pa
 
 # Set default NCCL parameters
 RUN echo NCCL_DEBUG=INFO >> /etc/nccl.conf
-
-ENV LD_LIBRARY_PATH=$OPEN_MPI_PATH/lib:$LD_LIBRARY_PATH
+RUN echo "pml = ob1" >> /opt/amazon/openmpi/etc/openmpi-mca-params.conf
```

Comment:
nit: [ADDRESS LATER] Use `OPEN_MPI_PATH` because we have already assigned `/opt/amazon/openmpi` to an ARG.